tools/buildtools/packmol/README.md

```diff
@@ -0,0 +1,12 @@
+
+# Packmol for Galaxy
+
+# See the packmol user guide for more
+http://m3g.iqm.unicamp.br/packmol/userguide.shtml#more
+
+# calculating the number of molecules ise the volume guesser @
+http://m3g.iqm.unicamp.br/packmol/utilities.shtml
+
+# run test
+planemo test --conda_dependency_resolution --conda_prefix /scratch/cbarnett/conda_for_planemo/ galaxy_packmol.xml
```

Comment:
--conda_dependency_resuolution should be the default by now.

I would remove this file I think